backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/shippingNotification/shippingNotification.feature

```diff
@@ -51,8 +51,19 @@ Feature: Shipping Notifications
     And metasfresh contains C_Auction:
       | Identifier | Name     | Date       |
       | Auction_1  | Auction1 | 2023-09-21 |
+    And load C_Element:
+      | C_Element_ID.Identifier | OPT.C_Element_ID |
+      | element_1               | 1000000          |
+    And load C_ElementValue:
+      | C_ElementValue_ID.Identifier | C_Element_ID.Identifier | Value |
+      | elementValue_1               | element_1               | 90000 |
+      | elementValue_2               | element_1               | 90055 |
```

Comment:
use P_ExternallyOwnedStock_Acct as the identifier

(i.e. something which is human/tester friendly, easy to read and understand)